hedera-node/src/main/java/com/hedera/services/sigs/sourcing/SigMapPubKeyToSigBytes.java

```diff
@@ -53,15 +50,18 @@
 
 	@Override
 	public byte[] sigBytesFor(byte[] pubKey) throws KeyPrefixMismatchException {
-		List<byte[]> matchingSigs = sigMap.getSigPairList()
-				.stream()
-				.filter(sp -> beginsWith(pubKey, sp.getPubKeyPrefix().toByteArray()))
-				.map(this::sigBytesFor)
-				.collect(toList());
-		if (matchingSigs.size() > 1) {
-			throw new KeyPrefixMismatchException("Source signature map is ambiguous for given public key!");
+		byte[] sigBytes = EMPTY_SIG;
```

Comment:
It would be better to have tests for `sigBytesFor`

Reply:
@abhishek-hedera  All the test cases are covered in existing tests in `PlatformSigOpsTest` and is used in `HederaToPlatformSigOpsTest` as well. So, I did not add extra test cases.